runtime-library/CMakeLists.txt

```diff
@@ -1,24 +1,31 @@
 # Copyright 2023 The IREE Authors
 #
-# Licensed under the Apache License v2.0 with LLVM Exceptions.
-# See https://llvm.org/LICENSE.txt for license information.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Licensed under the Apache License v2.0 with LLVM Exceptions. See
+# https://llvm.org/LICENSE.txt for license information. SPDX-License-Identifier:
+# Apache-2.0 WITH LLVM-exception
 
 cmake_minimum_required(VERSION 3.21...3.24)
 
+# NOTE: This file is for building the IREE runtime for Apple
+# platforms.  To build the IREE compiler for macOS, please use the
+# CMakeLists.txt file in the IREE source tree.
 project(IREE_RUNTIME_LIBRARY)
 cmake_policy(SET CMP0069 NEW)
 
-set(IREE_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../iree" CACHE STRING "Main IREE source")
+# NOTE: The IREE source code must be cloned to a directory side-by-side with
+# this project.
+set(IREE_ROOT_DIR
+    "${CMAKE_CURRENT_SOURCE_DIR}/../iree"
+    CACHE STRING "Main IREE source")
 
 # Enable LTO if supported.
 option(IREERT_ENABLE_LTO "Enable LTO (link time optimization) if supported" ON)
 if(APPLE)
   # Enabling LTO on macOS/iOS is fine to build frameworks. But when we merge
-  # them into an XCframework using the command
-  # xcodebuild -create-xcframework -framework build-ios-sim/ireert.framework ...
-  # it will complain "unable to find any architecture information in the binary",
-  # even though lipo -info could recognize the architecture of each framework.
+  # them into an XCframework using the command xcodebuild -create-xcframework
```

Comment:
Nit: can we start command on new line?

Reply:
Moved the command-line example in a new line.